### PR TITLE
add models

### PR DESCRIPTION
**Pull request by moklidia**

### CODE REVIEW

File: test/factories/tasks.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :task do
+    name { "MyString" }
```
Comment (artempartos) [inline test/factories/tasks.rb]: it's better (cleaner) to define sequences in /test/factories/sequences.rb and use them in the factories

File: test/models/admin_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class AdminTest < ActiveSupport::TestCase
```
Comment (artempartos) [inline test/models/admin_test.rb]: just remove model tests, we will not use them, we are testing through endpoints (controllers)

Comment (moklidia) [reply test/models/admin_test.rb]: Done!

File: test/factories/tasks.rb (hunk 1 above)
Comment (moklidia) [reply test/factories/tasks.rb]: Sequences added. I should write several sequences for each model with different attributes for it to generate several users/tasks for testing, right?

Comment (artempartos) [reply test/factories/tasks.rb]: In general your factory should be clean, like this
```
FactoryBot.define do
  factory :task do
    name
    description
    author { nil }
    assignee { nil }
    expired_at
  end
end
```

Comment (artempartos) [reply test/factories/tasks.rb]: State should we defined via state machine initial state; name, description and expired should be from sequences, all association nil by default. You also can define factories traits if you need task in some state or with defined association

File: test/factories/sequences.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+FactoryBot.define do
+  sequence :task do
```
Comment (artempartos) [inline test/factories/sequences.rb]: in sequnces you should have only data, dependent on postgres seqence. It means that postgres will generate some uniq number and you will have something like 'Name#{s}' = Name1, NameN - always uniq

File: test/factories/tasks.rb (hunk 1 above)
Comment (moklidia) [reply test/factories/tasks.rb]: Yep, now I finally get it. I've updated factories and sequences. I used traits to create different types of users, it that alright?

**Comment by artempartos** [comment]
👍

**Pull request merged**